libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/LoginActivity.java

```diff
@@ -100,6 +99,25 @@
 	@Override
 	protected void onCreate(Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
+		boolean isDarkTheme = SalesforceSDKManager.getInstance().isDarkTheme(this);
+        setTheme(isDarkTheme ? R.style.SalesforceSDK_Dark_Login : R.style.SalesforceSDK);
+        // This covers the case where OS dark theme is true, but app has disabled.
+        // TODO: Remove SalesforceSDK_AccessibleNav style when min API becomes 26.
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
This logic appears to be repeated in all activities. Could we move this to a util method somewhere? Maybe create a util class called `ThemeChooser` or something?